Gopkg.toml

```diff
@@ -137,7 +137,7 @@ required = [
 
 [[constraint]]
   name = "github.com/cloudevents/sdk-go"
-  version = "1.0.0"
+  revision = "427e8cf75237fdc7d4bb6c496398d7288efb8921"
```

Comment:
@n3wscott   I guess, based on 

https://github.com/knative/eventing/pull/2589#pullrequestreview-360121978


this is than ultimately required, right ?

Reply:
until there is a release, yeah this is required.